git2pantheon/api/upload.py

```diff
@@ -265,3 +270,52 @@ def progress_update_resources():
     return jsonify(
         response_dict
     ), 200
+
+
+@swag_from(get_docs_path_for('cache_clear_api.yaml'))
+@api_blueprint.route('/cache/clear', methods=['POST'])
+def clear_cache():
+    data = get_request_data()
+    cache_clear_result = {
+        "drupal_result_assemblies":{},
+        "drupal_result_modules": {}
+    }
+
+    try:
+        clear_drupal_cache(data, cache_clear_result)
+        clear_akamai_cache(data,cache_clear_result)
+    except Exception as e:
+        logger.error("Exception occurred while trying to clear  cache with error=" + str(e))
+        raise ApiError("Upstream Server Error", 503, details=str(e))
+    return jsonify(cache_clear_result)
+
+
+def drupal_cache_clear_bulk(cache_clear_result, cache_req_data):
```

Comment:
Can we add bulk in prefix like "bulk_cache_clear_drupal"
similar for "individual_cache_clear_drupal"